how_to_ask_for_help.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asking for help</title>
+</head>
+<body>
+    <h1>How to get Coding Help</h1>  
+    <p>This is a guide on how to get help when you run into a problem while coding.</p>
+    <p>Most people are very keen to offer help if they can, but I have noticed a few problems people make that mean it is harder for others to help them. This might cause you to get frustrated, and not get the help you need.</p>
+
+    <ol>
+        <li>
+            <h2>Let us see your code</h2>
+            <p>We are probably going to want to see the code you have typed, and try changing things. Sometimes it is not clear, right away, what is wrong. We want to edit the code and add logs and alter things. </p>
+            <ul>
+                <li>Put the code in a format we can edit. Screenshots or photos of the screen will not work for this, as people would have to type out all of your code just to start helping.</li>
+                <li>Include all of your code. If you just put part of the code that you think is the problem, we cannot help if it turns out the problem is in another part of your code. This is a big issue in CSS, where other parts of your CSS file can change things everywhere.</li>
+                <li>If you change something, post the updated code. This helps people see if you have made error when you have made corrections.</li>
+                <li>Use sites like <a href="https://github.com/">Github</a> or <a href="https://jsfiddle.net/">jsFiddle</a> to post your code in a way that people can see and work with it.</li>
+            </ul>
+        </li>
+        <li>
+            <h2>Tell us what you have tried</h2>
```

Comment:
I think it looks a little strange here to have this header, but no explanation.  Maybe add a sentence or two here expanding on this?